cpp/README.md

```diff
@@ -16,9 +16,9 @@ Directory structure:
 MEmilio C++ uses CMake as a build configuration system (https://cmake.org/)
 
 MEmilio C++ is regularly tested with the following compilers (list will be extended over time):
-- GCC, versions 7.3.0 - 10.2.0
-- Clang, version 9.0
-- MSVC, versions 19.16.27045.0 (Visual Studio 2017) - 19.29.30133.0 (Visual Studio 2019)
+- GCC, versions 9.4.0 - 11.4.0
+- Clang, version 6.0.1 - 14.0.0
+- MSVC, versions 19.29.30154.0 (Visual Studio 2019) - 19.38.33135.0 (Visual Studio 2022)

```

Comment:
```suggestion
- GCC, versions 9.4.0 and 11.4.0
- Clang, version 6.0.1 and 14.0.0
- MSVC, versions 19.29.30154.0 (Visual Studio 2019) and 19.38.33135.0 (Visual Studio 2022)
```
The dash looks like it indicates a range, I think we should change it (a comma would also be fine).

Reply:
Is it ensured that the subversions do always stay the same? Maybe not? We only ask for gcc-11 for example. Maybe we should write:

`GCC, versions 9 and 11 (versions 9.4.0 and 11.4.0 as of May 07, 2024)`
?

Reply:
Good point, I did not consider the subversions.

Though, I don't think they matter too much, it might be fine to just omit them, since they can get outdated quickly. There were two minor msvc updates I noticed recently, and I don't even have it installed.

Reply:
I think it makes sense to name at least the first number after the dot. There are sometimes significant changes from X.Y to X.Y+1